### PR TITLE
#15441 Repro: Settings menu items order is reversed in EE version

### PR DESCRIPTION
**Pull request by nemanjaglumac**
### Status
PENDING REVIEW

### What does this PR accomplish?
- Reproduces #15441 

### How to test this manually?
- `yarn test-cypress-open --spec frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js`
- All tests should pass

### Additional notes:
- The bug was fixed in #15607

### Screenshots:
image

### CODE REVIEW

File: frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js [hunk 1]
```diff
@@ -325,6 +325,17 @@ describe("scenarios > admin > settings", () => {
...
 
+  it("should display the order of the settings items consistently between OSS/EE versions (metabase#15441)", () => {
+    cy.visit("/admin/settings/setup");
+    cy.get(".AdminList .AdminList-item")
+      .as("settingsOptions")
+      .first()
+      .contains("Setup");
+    cy.get("@settingsOptions")
+      .last()
+      .contains("Whitelabel");
```
Comment (flamber) [inline frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js:336]: That will fail on OSS. Whilelabel is an EE feature.

Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js:336]: Does it make sense to check for ALL members of the array and their position in UI?

Comment (nemanjaglumac) [reply frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js:336]: I'll fix this, btw.
Great catch - I totally overlooked it.

Comment (flamber) [reply frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js:336]: No, keep it semi-flexible, but it's the second time that Whitelabel has ended up on top, so a test for first+last seems good.

File: frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js [hunk 2]
```diff
@@ -325,6 +325,21 @@ describe("scenarios > admin > settings", () => {
     cy.findByText(/Site URL/i);
   });
 
+  it("should display the order of the settings items consistently between OSS/EE versions (metabase#15441)", () => {
+    const lastItem = Cypress.env("HAS_ENTERPRISE_TOKEN")
+      ? "Whitelabel"
+      : "Caching";
```
Comment (nemanjaglumac) [inline frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js:331]: @flamber this should do it.

**Review by flamber — APPROVED** [review]

**Pull request merged**